docs/content/standards/coin.mdx

```diff
@@ -0,0 +1,305 @@
+---
+title: Coin Standard
+description: The Sui Coin standard enables you to create a broad range of fungible tokens on the Sui network to satisfy a number of use cases. The Coin standed on Sui is equivalent to the ERC-20 technical standard on Ethereum. 
+sidebar_label: Coin
+---
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+The Coin standard is the technical standard used for smart contracts on Sui for creating coins on the Sui blockchain. The standardization of coin creation on Sui means that wallets, exchanges, and other smart contracts can manage coins created on Sui the same as they manage SUI, without any additional processing logic. 
+
+See [Sui Tokenomics](../concepts/tokenomics.mdx) to learn more about the SUI native coin and its use on the Sui network.
+
+Although coins on Sui follow the Coin standard, they can offer specialized abilities. For example, you can create a regulated token that allows its creator to add specific addresses to a deny list, so that the identified addresses cannot use the token as inputs to transactions.
+
+See the [coin module](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/docs/sui-framework/coin.md) documentation for all available options when creating a coin-type token on Sui. 
+
+## Fungible tokens
+
+Coins on Sui are fungible tokens by definition, which means there are many of a specific type (`Coin`) and each token of that type is the same as the next. If you created a regulated coin and named it REGCOIN, then you could mint as many of those coins as you wanted and each REGCOIN would have the same properties as the next.
```

Comment:
Just attempting to define fungible. Will reword.

Reply:
Ah got it, the definition for me is that every unit of `Coin<C>` is indistinguishable from every other unit (they don't each have their own identity) which means they can be freely merged together and split apart within individual instances of the `Coin<C>` type.

The important distinction is that this property is only true between instances of `Coin<C>` rather than instantiations of `Coin` (the `C` matters), and the property is really to do with the thing that is counted in the balance of a `Coin<C>`. What I mean by that is that two instances of `Coin<C>` are not fungible with each other, because one could have a balance of `1000000` and another could have a balance of `1`, but if I took one unit from the million-balance `Coin<C>`, that would be indistinguishable from the other unit coin.